translations/README-he.md

```diff
@@ -53,18 +52,26 @@
     </p>
     <hr />
 </div>
+<div dir='rtl'>
+	<h2 name='what-is-it'> על מה מדובר כאן?</h2>
+	<img src='https://camo.githubusercontent.com/223e9758b09eb2908cb04c01c05f6e8550596dfd1526db5a2895c471b378a538/68747470733a2f2f64336a32706b6d6a74696e366f752e636c6f756466726f6e742e6e65742f636f64696e672d61742d7468652d7768697465626f6172642d73696c69636f6e2d76616c6c65792e706e67' alt='software engineer explains using a white board'/>
```

Comment:
Let's keep the image URLs the same as on the English readme. This way they can be updated in one place.